modules/base/admin/module.py

```diff
@@ -645,5 +646,5 @@ async def spamchannel_primary(self, ctx, channel: discord.TextChannel):
         )
 
 
-async def setup(bot) -> None:
+async def setup(bot: Strawberry) -> None:
```

Comment:
Documentation

Reply:
I think here it's not really necessary, see: https://discordpy.readthedocs.io/en/latest/ext/commands/api.html?highlight=load_extension#discord.ext.commands.Bot.load_extension